build/commands/scripts/iosCommands.js

```diff
@@ -39,7 +39,7 @@ const bootstrap = (options = {}) => {
   if (options.force) {
     bootstrapArgs.push('--force')
   }
-  util.run('vpython3', bootstrapArgs, config)
+  util.run('python3', bootstrapArgs, config.defaultOptions)
```

Comment:
pls do the same change in `brave/DEPS`. It's better to run the script using the same interpreter everywhere.

Reply:
@goodov Done!

Reply:
The actual change required is `config` → `config.defaultOptions`, which adds the correct paths to the call (otherwise depot_tools paths are not in there)

The swap to `python3` over `vpython3` was just because most commands we have use `python3` instead of `vpython3`